src/list-grid.scss

```diff
@@ -0,0 +1,232 @@
+        left: 0;
+        margin: auto !important;
+      }
+
+      &--state-unread {
+        .fd-list__item-body * {
+          font-weight: bold;
+        }
+      }
+
+      &--state-actions {
+        margin-top: 1rem;
+      }
+
+      &--link {
+        @include fd-navigated() {
+          &::after {
+            content: "";
+            position: absolute;
+            top: 0;
+            right: 0;
+            height: 100%;
+            width: 0.1875rem;
+            display: block;
+            border-radius: 0 $fd-grid-list-border-radius $fd-grid-list-border-radius 0;
+            background-color: var(--sapList_SelectionBorderColor);
+          }
+        }
+      }
+    }
+
+    &__link {
+      flex-direction: column;
+      align-items: normal;
+      padding: 0;
+
+      @include fd-active() {
+        * {
+          color: $fd-grid-list-active-text-color !important;
+        }
+      }
+    }
+
+    &__group-header {
+      font-size: var(--sapFontHeader5Size);
+      padding-top: 0.75rem !important;
+      background-color: transparent;
+      border: 0;
+      margin-top: 1rem;
+      height: 2.75rem;
+
+      &:first-child {
+        margin-top: 0 !important;
+      }
+    }
+  }
+
+  &__mode {
+    &--single-select-left,
+    &--multi-select {
+      .fd-list__item {
+        .fd-radio + .fd-radio__label,
+        .fd-checkbox + .fd-checkbox__label {
+          display: flex;
+          align-self: inherit;
+          color: var(--sapContent_LabelColor);
+        }
+      }
+    }
+
+    &--single-select-master,
+    &--multi-select,
+    &--single-select-left {
+      .fd-list__item {
+        cursor: default !important;
+
+        @include fd-selected() {
+          box-shadow: $fd-grid-list-mode-box-shadow;
+
+          .fd-list__link {
+            @include fd-hover() {
+              box-shadow: $fd-grid-list-mode-box-shadow;
+              background: var(--sapList_SelectionBackgroundColor);
+            }
+
+            @include fd-active() {
+              background-color: var(--sapList_Active_Background);
+            }
+          }
+
+          @include fd-hover() {
+            box-shadow: $fd-grid-list-mode-box-shadow;
+            background: var(--sapList_SelectionBackgroundColor) !important;
+          }
+        }
+
+        @include fd-hover() {
+          background-color: var(--sapList_Background) !important;
+        }
+      }
+    }
+
+    &--single-select-master {
+      .fd-list__item {
+        cursor: pointer !important;
+
+        @include fd-selected() {
+          cursor: default !important;
+        }
+
+        @include fd-hover() {
+          box-shadow: var(--fdGrid_List_SingleSelect_Hover_BoxShadow);
+        }
+      }
+    }
+  }
+
+  &__more {
+    cursor: pointer;
+    padding: 0.813rem;
+    background-color: var(--sapGroup_ContentBackground);
+    border-radius: $fd-grid-list-border-radius;
+    box-shadow: $fd-grid-list-item-box-shadow;
+    text-align: center;
+
+    h6 {
```

Comment:
Can you remove all the element selectors?

Reply:
+1

Reply:
fixed